### ISSUE TITLE
Pagination never terminates for oci.tenant_manager_control_plane.OrganizationClient.list_organizations in Python SDK

### ISSUE THREAD

**Issue by davide-maestroni** [oracle/oci-python-sdk#718]
It looks like pagination is broken for oci.tenant_manager_control_plane.OrganizationClient.list_organizations, each call returns a response object with a different next_page token and has_next_page set to True, but the items in the data are always the same.

To reproduce just run:

organization_client = oci.tenant_manager_control_plane.OrganizationClient(config)
for response in oci.pagination.list_call_get_all_results_generator(
    organization_client.list_organizations, "response", tenancy_ocid
):
    print(response.data.items)
    print(response.next_page)
    print(response.has_next_page)

**Comment by adizohar** [comment]
Hi, Did you try the below ?
```
organization_client = oci.tenant_manager_control_plane.OrganizationClient(config)
result = oci.pagination.list_call_get_all_results(organization_client.list_organizations, tenant_id)
print(result.data)
```

**Comment by davide-maestroni** [comment]
Yes, I was initially doing that, but the call to oci.pagination.list_call_get_all_results never exited.
After some debugging I realized that the pagination mechanism was broken.

**Comment by adizohar** [comment]
Hi Davide, how many organizations do you have in order to check the pagination ?

**Comment by davide-maestroni** [comment]
In my testing setup there is just one Tenancy and one Organization.

**Comment by adizohar** [comment]
Without pagination you will be able to extract all the organizations that allowed
I don't see any issue here, once you have 100+ organizations and work with the pagination API, let us know if any issue

**Comment by davide-maestroni** [comment]
@adizohar, now the pagination is behaving correctly. I guess something has been fixed on the backend side.
This is exactly what I was hoping for.
Thank you for the support
